java/fury-core/src/main/java/org/apache/fury/resolver/ClassResolver.java

```diff
@@ -305,33 +304,26 @@ public void initialize() {
 
   private void addDefaultSerializers() {
     // primitive types will be boxed.
-    addDefaultSerializer(String.class, new StringSerializer(fury));
+    StringSerializer.registerDefaultSerializers(fury);
     PrimitiveSerializers.registerDefaultSerializers(fury);
     Serializers.registerDefaultSerializers(fury);
     ArraySerializers.registerDefaultSerializers(fury);
     TimeSerializers.registerDefaultSerializers(fury);
     OptionalSerializers.registerDefaultSerializers(fury);
     CollectionSerializers.registerDefaultSerializers(fury);
     MapSerializers.registerDefaultSerializers(fury);
-    addDefaultSerializer(Locale.class, new LocaleSerializer(fury));
```

Comment:
those files has only one serializer, maybe it's OK to register such serializers directly here?

Reply:
Yes, but it may be better in design to move them into their own Serializers, which are only responsible for calling their respective `Serializer#registerDefaultSerializers`.

Reply:
It has only one serializer, I pprefer not using plurality.

Reply:
And the reason`registerDefaultSerializers` exists is becuase there are multiple deeply-related serializers in one file

Reply:
Ok, get it.